### DISCUSSION TITLE
Latest check-in to main branch broke USB memory stick access

### DISCUSSION THREAD

**Discussion by vruumllc** [picocomputer/community#47] [Q&A] [answered]
Hi,
I tried pulling, building, and installing the latest from the master branch.
Although CapsLock indeed worked on my USB wireless keyboard, I could no longer see or access my USB memory stick.
I reverted back to 0.2 release, and my USB stick access is fine again.

**Comment by rumbledethumps** [comment]
I haven't gone anywhere near MSC code in a very long time. Try building the 0.2 release (it's tagged). USB MSC is definitely working for me. Also, look at the USB devices in STATUS.

**Reply by vruumllc** [reply]
Here's a clue, maybe:

My USB stick normally enumerates at a higher number than my keyboard. If I move the USB stick so it enumerates lower than the keyboard, then I can (after some delay) see the USB stick with the new code.

![image0](https://github.com/picocomputer/discussions/assets/145504135/4fb102be-6b65-40ba-99ab-d9106d4ccc3f)
![image2](https://github.com/picocomputer/discussions/assets/145504135/6b3f7698-5f3b-46d3-9fd7-a1b7f3659297)
![image1](https://github.com/picocomputer/discussions/assets/145504135/87434085-bd96-4c3c-815c-7200551a00cb)

Also, note that it no longer reports mouse for my dual-device wireless dongle, although that may be expected???

**Reply by rumbledethumps** [reply]
Try now. I'm guessing since there's no docs and no examples.

**Reply by vruumllc** [reply, marked as answer]
Works great now. That was fast. Good job!
Mouse also shows up. (Mouse Events no longer printed to display, but I assume that is by design).

**Reply by vruumllc** [reply]
Yay! My Favorite Anker wireless keyboard (and mouse, I assume) are also working well now.
Previously keyboard was causing spurious 0 --- events to be displayed on screen, messing up output.
